ui/src/app/applications/components/application-status-panel/application-status-panel.tsx

```diff
@@ -127,12 +129,12 @@ export const ApplicationStatusPanel = ({application, showOperation, showConditio
                             </a>
                             {appOperationState.syncResult && appOperationState.syncResult.revision && (
                                 <div className='application-status-panel__item-value__revision show-for-large'>
-                                    To <Revision repoUrl={source.repoURL} revision={appOperationState.syncResult.revision} />
+                                    to <Revision repoUrl={source.repoURL} revision={appOperationState.syncResult.revision} />
                                 </div>
                             )}
                         </div>
 
-                        <div className='application-status-panel__item-name'>
+                        <div className='application-status-panel__item-name' style={{marginBottom: '0.5em'}}>
                             {appOperationState.phase} <Timestamp date={appOperationState.finishedAt || appOperationState.startedAt} />
```

Comment:
I suppose adding the "When:" label is out of scope of this PR

Reply:
Yeah, I figured we can address this later when we address the restyling of buttons. Wanted to keep this PR small